src/main/java/com/iuc/requests/service/UserService.java

```diff
@@ -37,28 +37,29 @@
     return staffsDto;
   }
 
-  public Staff createStaff(StaffDto staffDto) {
+  public StaffDto createStaff(StaffDto staffDto) {
 
     try {
-      return staffRepository.save(modelMapper.map(staffDto, Staff.class));
+      return modelMapper.map(
+          staffRepository.save(modelMapper.map(staffDto, Staff.class)), StaffDto.class);
     } catch (Exception e) {
       return null;
     }
   }
 
-  public Staff findStaffByEmail(String email) {
+  public StaffDto findStaffByEmail(String email) {
 
     try {
-      return staffRepository.findByEmail(email);
+      return modelMapper.map(staffRepository.findByEmail(email), StaffDto.class);
     } catch (EntityNotFoundException e) {
```

Comment:
Which method usage throw EntityNotFoundException? I don't understand why you catch an exception